gen/cycle.ml

```diff
@@ -515,7 +526,9 @@ module CoSt = struct
          | Ord ->
             co_cell.(0) <- E.overwrite_value old e.atom cell2
          | Pair -> (* No Rmw for pairs *)
-            co_cell.(0) <- E.overwrite_value old e.atom (e.v-1);
+                 (* TODO check the variable name `width` *)
```

Comment:
what needs to be checked about the variable name?

Reply:
It is no longer a valid comment. I will remove it.